sushichef.py

```diff
@@ -142,10 +136,12 @@ def generate_child_topics(arvind_contents, main_topic, lang_obj, topic_type):
     # Create a topic for each languages
     pp = pprint.PrettyPrinter()
     data = arvind_contents[lang_obj.name]
+    standard_counter = 0
     for topic_index in data:
 
         if topic_type == STANDARD_TOPIC:
-            source_id = generate_source_id(lang_obj.code)
+            standard_counter += 1
+            source_id = 'arvind-{0}-child-topic-{1}'.format(lang_obj.name, standard_counter)
```

Comment:
If there's no unique identifier, then I would use the topic's title rather than a counter, as it is fairly unique and less likely to change than the sort order.

Reply:
I'll implement this @kollivier and remove the counter at the source id.